App.tsx

```diff
@@ -58,22 +60,24 @@ export default function App() {
   } else {
     return (
       <SafeAreaProvider>
-       {!signedIn && 
-       <View style={styles.containerSignIn}>
-          <Image style={styles.gradient0} width={'100%'} height={'70%'} source={require('./assets/images/signinbackgroundhue.png')} />
-          <View style={styles.innerContainer}>
-              <Image width={100} height={100} source={require('./assets/images/signinlogoblurry.png')}/>
-              <View style={styles.signInButton}>
-                  <Text onPress={handleGoogleSignIn} style={styles.signInText}>Sign in with Google</Text>
+        <Provider store={GlobalStore}>
```

Comment:
I think the redux provider should be outside (wrapped around) `SafeAreaProvider`. SafeAreaProvider is basically a view, and redux provider should contain the entire app. Feel free to do some more research to confirm this if you have time!

Some sources:
https://github.com/th3rdwave/react-native-safe-area-context#react-native-safe-area-context
https://stackoverflow.com/questions/49306334/react-navigation-apply-safeareaview-globally-with-redux

Reply:
I checked, and you're right! I fixed this.